src/textual_dev/previews/example_widgets/__init__.py

```diff
@@ -0,0 +1,158 @@
+import random
+from statistics import mean
+
+from textual.app import ComposeResult
+from textual.containers import Container, Center, Middle
+from textual.widgets import (
+    DirectoryTree,
+    Footer,
+    Header,
+    Input,
+    Label,
+    ListView,
+    ListItem,
+    LoadingIndicator,
+    Sparkline,
+    Static,
+    Tree,
+    ProgressBar,
+    TabbedContent,
+    TabPane,
+    Markdown,
+    Tabs,
+    TextLog,
+)
+
+from .button import button_example
+from .checkbox import checkbox_example
+from .content_switcher import content_switcher_example
+from .data_table import data_table_example
+from .markdown import markdown_viewer_example, markdown_example
+from .option_list import option_list_example
+from .placeholder import placeholder_example
+from .pretty import pretty_example
+from .radio import radio_button_example, radio_set_example
+from .select import select_example, selection_list_example
+from .switch import switch_example
+from .text_log import text_log_example
+
+
+def directory_tree_example(id: str) -> ComposeResult:
+    yield Container(DirectoryTree("./"), id=id)
+
+
+def footer_example(id: str) -> ComposeResult:
+    yield Container(Footer(), id=id)
+
+
+def header_example(id: str) -> ComposeResult:
+    yield Container(Header(), id=id)
+
+
+def input_example(id: str) -> ComposeResult:
+    yield Container(
+        Input(placeholder="First Name"), Input(placeholder="Last Name"), id=id
+    )
+
+
+def label_example(id: str) -> ComposeResult:
+    yield Container(Label("Hello, world!"), id=id)
+
+
+def list_item_example(id: str) -> ComposeResult:
+    yield Container(
+        ListView(
+            ListItem(Label("One")),
+            ListItem(Label("Two")),
+            ListItem(Label("Three")),
+        ),
+        id=id,
+    )
+
+    yield Footer()
+
+
+def loading_example(id: str) -> ComposeResult:
+    yield Container(LoadingIndicator(), id=id)
+
+
+def sparkline_example(id: str) -> ComposeResult:
+    data = [random.expovariate(1 / 3) for _ in range(1000)]
+
+    yield Container(
+        Sparkline(data, summary_function=max),
+        Sparkline(data, summary_function=mean),
+        Sparkline(data, summary_function=min),
+        id=id,
+    )
+
+
+def static_example(id: str) -> ComposeResult:
+    yield Container(Static("Hello, world!"), id=id)
+
+
+def tree_example(id: str) -> ComposeResult:
+    tree: Tree[dict] = Tree("Dune")
+    tree.root.expand()
+    characters = tree.root.add("Characters", expand=True)
+    characters.add_leaf("Paul")
+    characters.add_leaf("Jessica")
+    characters.add_leaf("Chani")
+    yield Container(tree, id=id)
+
+
+def progress_bar_example(id: str) -> ComposeResult:
+    bar = ProgressBar(total=100, show_eta=False)
+    bar.advance(50)
+    yield Container(Center(Middle(bar)), id=id)
```

Comment:
Given that the progress bar is static, which makes sense, maybe create it as `ProgressBar()` and don't advance it. That way, the user can see the indeterminate animation.

Reply:
Do I understand it correctly that you want to remove `bar.advance(50)`?

Reply:
Remove `bar.advance(50)` and also `total=100`.